sql/telemetry/sync/view.sql

```diff
@@ -1,4 +1,7 @@
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.sync`
-AS SELECT * FROM
+AS SELECT
+  * REPLACE(
+    `moz-fx-data-shared-prod.udf.normalize_metadata`(metadata) AS metadata)
```

Comment:
We had missed applying this change to the sync ping view (which we actually commit to the repo here due to the extra v5 version of this table).